tests/lib/rules/no-tabs.js

```diff
@@ -24,10 +24,13 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n" +
         "  //   sdfdsf \n" +
         "}",
+        "var str = 'test\tstring'",
+        "var re = /test\tstring/"
```

Comment:
Ah, sorry.
I wrote "should not modify ... automatically.", it's not meaning "should not warn".
The code above should be warned, but not be fixed automatically.

In short, the `fix` function should return `null` if the tab is in literals.